### PR TITLE
feat(vocabulary): Add the option to set 'single value', 'multi value'…

### PR DESCRIPTION
**Pull request by ioanpocol**
… and 'do not show' opyions for selection type (SDESK-3551)

### CODE REVIEW

File: scripts/apps/vocabularies/views/vocabulary-config-modal.html [hunk 1]
```diff
@@ -52,6 +52,15 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                          ng-show="vocabularyForm.$error.maxlength" translate>The helper text must not exceed 120 characters.</div>
                 </div>
 
+                <div class="sd-line-input" ng-if="matchFieldTypeToTab('vocabularies', vocabulary.field_type)">
+                    <label for="cv_selection_type" class="sd-line-input__label" translate>Show to users as:</label>
+                    <select class="input-medium" ng-model="vocabulary.selection_type">
+                        <option value="single selection" translate>single selection</option>
```
Comment (tomaskikutis) [inline scripts/apps/vocabularies/views/vocabulary-config-modal.html]: Create a constant with selection types. Iterate it in this select and use it for comparison in other places.

Comment (ioanpocol) [reply scripts/apps/vocabularies/views/vocabulary-config-modal.html]: Done

File: scripts/apps/vocabularies/views/vocabulary-config-modal.html [hunk 2]
```diff
@@ -52,6 +52,13 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                          ng-show="vocabularyForm.$error.maxlength" translate>The helper text must not exceed 120 characters.</div>
                 </div>
 
+                <div class="sd-line-input" ng-if="matchFieldTypeToTab('vocabularies', vocabulary.field_type)">
+                    <label for="cv_selection_type" class="sd-line-input__label" translate>Show to users as:</label>
```
Comment (tomaskikutis) [inline scripts/apps/vocabularies/views/vocabulary-config-modal.html]: `for` attribute only makes sense if there's another element with a corresponding ID. In this case, `select` element should have the id `cv_selection_type`

File: scripts/apps/vocabularies/constants.ts [hunk 3]
```diff
@@ -15,6 +15,21 @@ export const MEDIA_TYPES = {
...
+export const VOCABULARY_SELECTION_TYPES = {
+    SINGLE_SELECTION: {
+        id: 'single selection',
+        label: gettext('Single selection')
+    },
+    MULTIPLE_SELECTION: {
+        id: 'multi selection',
+        label: gettext('Multi selection')
+    },
+    DO_NOT_SHOW: {
```
Comment (tomaskikutis) [inline scripts/apps/vocabularies/constants.ts]: I can't find any usages of this. Is it needed?

Comment (ioanpocol) [reply scripts/apps/vocabularies/constants.ts]: Yes, there is a task about it:
https://dev.sourcefabric.org/browse/SDESK-3625

Comment (tomaskikutis) [reply scripts/apps/vocabularies/constants.ts]: Are going to implement this in another PR or should it be working right now?

Comment (ioanpocol) [reply scripts/apps/vocabularies/constants.ts]: The task is still in TODO, so the fix will be in a different PR.

File: scripts/apps/vocabularies/views/vocabulary-config-modal.html (hunk 2 above)
Comment (ioanpocol) [reply scripts/apps/vocabularies/views/vocabulary-config-modal.html]: Done

Comment (tomaskikutis) [reply scripts/apps/vocabularies/views/vocabulary-config-modal.html]: Please also fix the issue I mentioned in the review comment

**Comment by ioanpocol** [comment]
>scripts/core/notification/notification.js:118 - Put data.vocabulary outside of gettext since the translation will never get matched if it's inside. I know you haven't changed this part, but let's fix it anyway.

Done

**Pull request merged**